front/src/components/App/SalesArea/SalesArea.tsx

```diff
@@ -2,37 +2,80 @@ import Button from "../../UI/Button/Button";
 import Header from "../../UI/Header/Header";
 import RouteLink from "../../UI/RouteLink/RouteLink.styled";
 import { APP_PAGE } from "../App";
-import allegro_image from "../../Images/allegro.jpg";
-import olx_image from "../../Images/olx.jpg";
-import ebay_image from "../../Images/ebay.jpg";
-import CompaniesSector from "../../UI/CompanyView/CompaniesSector";
+import sony_image from "../../Images/sony.jpg";
+import jbl_image from "../../Images/jbl.jpg";
+import steelseries_image from "../../Images/myszka.jpg";
+import Products from "../../UI/ProductView/ProductSector";
 import { SalesAreaStyled } from "./SalesArea.styled";
+import Auctions from "../../UI/AuctionView/AuctionSector";
+import React, { useState } from "react";
+import ProductFilter from "../../UI/ProductView/ProductFilter";
 
-const EXAMPLES_COMPANIES = [
+const EXAMPLES_PRODUCTS = [
   {
     key: "1",
-    title: "Allegro",
-    img_src: allegro_image,
+    title: "TV Sony",
+    img_src: sony_image,
   },
   {
     key: "2",
-    title: "OLX",
-    img_src: olx_image,
+    title: "Słuchawki JBL",
+    img_src: jbl_image,
   },
   {
     key: "3",
-    title: "eBay",
-    img_src: ebay_image,
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
+  },
+];
+
+const EXAMPLES_AUCTIONS = [
+  {
+    key: "1",
+    auction_site: "Allegro",
+    title: "TV Sony",
+    img_src: sony_image,
+  },
+  {
+    key: "2",
+    auction_site: "Allegro",
+    title: "Słuchawki JBL",
+    img_src: jbl_image,
+  },
+  {
+    key: "3",
+    auction_site: "Allegro",
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
+  },
+  {
+    key: "4",
+    auction_site: "OLX",
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
   },
 ];
 
 const SalesArea = () => {
+  const [value, setValue] = useState("Products");
+
+  const handleChange = (selected: string) => {
+    setValue(selected);
+  };
+
   return (
     <SalesAreaStyled>
+      <Header />
+      <ProductFilter value={value} onChangeFilter={handleChange} />
+      {(value === "Products" || value === "Both") && (
+        <Products items={EXAMPLES_PRODUCTS} />
```

Comment:
tak żeby bardziej zasymulować że te dane przyszły z backendu to oba EXAMPLES potraktuj jedynie jako wartości inicjalne. Do tych komponentów przekazuj już wartości trzymane np w useState